### PR TITLE
Recruiting UI improvements

### PR DESCRIPTION
**Pull request by sumboid**

### CODE REVIEW

File: packages/ui/src/components/IconGroup.svelte [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+// 
+// See the License for the specific language governing permissions and
+// limitations under the License.
+-->
+<script lang="ts">
+  import type { IntlString, UIComponent } from '@anticrm/status'
+
+  import Tooltip from './Tooltip.svelte'
+
+  interface Item {
```
Comment (haiodo) [inline packages/ui/src/components/IconGroup.svelte:20]: @sumboid Please check if ui/types.ts had some similar, if not put into UI please.

File: packages/ui/src/index.ts [hunk 2]
```diff
@@ -13,7 +13,7 @@
 // limitations under the License.
 //
 
-import { UIComponent } from 'svelte'
+import { UIComponent } from '@anticrm/status'
```
Comment (haiodo) [inline packages/ui/src/index.ts:16]: I supposed this change was already merged?

File: plugins/calendar-impl/src/components/EditEvent.svelte [hunk 3]
```diff
@@ -17,15 +17,16 @@
   import type { DerivedEvent, Event } from '@anticrm/calendar'
   import calendar from '@anticrm/calendar'
   import type { QueryUpdater } from '@anticrm/presentation'
-  import { getClient } from '@anticrm/workbench'
+  import { getClient, WorkbenchRoute } from '@anticrm/workbench'
```
Comment (haiodo) [inline plugins/calendar-impl/src/components/EditEvent.svelte:20]: Please use type import for WorkbenchRoute

File: plugins/calendar-impl/src/components/EventTable.svelte [hunk 4]
```diff
@@ -66,6 +65,10 @@
 
   $: data = events.concat(dEvents)
   $: console.log(data)
```
Comment (haiodo) [inline plugins/calendar-impl/src/components/EventTable.svelte:67]: Could we remove console.log?

File: packages/ui/src/index.ts (hunk 2 above)
Comment (sumboid) [reply packages/ui/src/index.ts:16]: Yeah, will disappear after rebase

**Review by haiodo — APPROVED** [review]

**Pull request merged**